[2021-06-28 08:33:45,994] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 2021-06-28T08:33:40.896547+00:00 [queued]>
[2021-06-28 08:33:46,392] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 2021-06-28T08:33:40.896547+00:00 [queued]>
[2021-06-28 08:33:46,432] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-28 08:33:46,434] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-06-28 08:33:46,458] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-28 08:33:46,700] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_3> on 2021-06-28T08:33:40.896547+00:00
[2021-06-28 08:33:46,740] {standard_task_runner.py:52} INFO - Started process 3082 to run task
[2021-06-28 08:33:46,859] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'trigger_rule', 'task_3', '2021-06-28T08:33:40.896547+00:00', '--job-id', '480', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/trigger_rule.py', '--cfg-path', '/tmp/tmpvz_vj0zp', '--error-file', '/tmp/tmpj2h95dnh']
[2021-06-28 08:33:46,860] {standard_task_runner.py:77} INFO - Job 480: Subtask task_3
[2021-06-28 08:33:47,354] {logging_mixin.py:104} INFO - Running <TaskInstance: trigger_rule.task_3 2021-06-28T08:33:40.896547+00:00 [running]> on host airflowvm
[2021-06-28 08:33:48,111] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=trigger_rule
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2021-06-28T08:33:40.896547+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-28T08:33:40.896547+00:00
[2021-06-28 08:33:48,114] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-06-28 08:33:48,118] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'exit 0']
[2021-06-28 08:33:48,141] {subprocess.py:75} INFO - Output:
[2021-06-28 08:33:48,146] {subprocess.py:83} INFO - Command exited with return code 0
[2021-06-28 08:33:48,247] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=trigger_rule, task_id=task_3, execution_date=20210628T083340, start_date=20210628T083346, end_date=20210628T083348
[2021-06-28 08:33:48,438] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-28 08:33:48,491] {local_task_job.py:151} INFO - Task exited with return code 0
